recipe/meta.yaml

```diff
@@ -170,6 +170,13 @@ requirements:
     - {{ pin_compatible("nss") }}        # [unix]
     - {{ pin_compatible("nspr") }}       # [unix]
     - libxcb                             # [linux]
+    # Adding the following xcb-util run requirements resolves issue
+    # https://github.com/conda-forge/qt-main-feedstock/issues/32
+    - xcb-util                           # [linux and not aarch64]
+    - xcb-util-wm                        # [linux and not aarch64]
+    - xcb-util-keysyms                   # [linux and not aarch64]
+    - xcb-util-image                     # [linux and not aarch64]
+    - xcb-util-renderutil                # [linux and not aarch64]
```

Comment:
Why are these dependencies not added for `aarch64`?

Reply:
See comment above: https://github.com/conda-forge/qt-main-feedstock/pull/39#issuecomment-1152520217 But thanks to @hmaarrfk they are marked for migration.